server/app/modules/DatabaseSeedModule.java

```diff
@@ -33,7 +33,9 @@ public DatabaseSeedScheduler(
               // with Play evolutions. Evolutions must run before we seed database.
               // It doesn't seem to be a way to run code after evolutions so just
               // give them few sec to run.
-              Duration.ofSeconds(5), () -> databaseSeedTaskProvider.get().run(), executionContext);
```

Comment:
Here and below, `run()` just called `seedQuestions()` anyways; this is more clear.